compiler/rustc_span/src/lib.rs

```diff
@@ -2200,6 +2200,7 @@ pub struct ErrorGuaranteed(());
 impl ErrorGuaranteed {
     /// To be used only if you really know what you are doing... ideally, we would find a way to
     /// eliminate all calls to this method.
+    #[deprecated = "`Handler::delay_span_bug` should be preferred over this function"]
```

Comment:
can you instead make it say `Session::delay_span_bug`? Because that's what people will actually use.

Reply:
lol i had originally written session here